doc/pages/keys.asciidoc

```diff
@@ -367,6 +327,61 @@ the Shift modifier and moving will extend each selection instead.
 *<a-(>*::
     rotate selections content backward
 
+== Changes through external programs
+
+Shell expansions are available, (See <<expansions#shell-expansions,`:doc expansions shell-expansions`>>)
+Piping uses the *|* register by default (See <<registers#,`:doc registers`>>)
```

Comment:
It wasn't immediately obvious to me what piping would use the `|` register *for*; my first guess was that the pipe output would be stored in the register for later pasting.

Perhaps "The default command comes from the `|` register by default..."